src/components/DatePicker.js

```diff
@@ -0,0 +1,454 @@
+        top: '50%',
+        transform: 'translateY(-50%)'
+      },
+      calendarShow: {
+        display: 'block'
+      },
+      clearFix: {
+        clear: 'both'
+      },
+      component: {
+        backgroundColor: '#FFFFFF',
+        color: '#000000',
+        fontSize: this.props.fontSize,
+        height: '100%',
+        WebkitAppearance: 'none',
+        width: '100%',
+
+        '@media (max-width: 768px)': {
+          fontSize: '0.6em'
+        },
+
+        ':focus': {
+          boxShadow: 'none',
+          outline: 'none'
+        }
+      },
+      componentBottom: {
+        backgroundColor: '#FFFFFF',
+        borderBottomLeftRadius: '3px',
+        borderBottomRightRadius: '3px',
+        borderColor: '#E5E5E5',
+        borderStyle: 'solid',
+        borderWidth: '1px 1px 1px 1px',
+        boxShadow: '0 30px 30px 10px rgba(0,0,0,0.1)',
+        boxSizing: 'border-box',
+        display: 'none',
+        marginTop: '10px',
+        maxWidth: '300px',
+        padding: '0px 0px 0px 0px',
+        position: 'absolute',
+        width: '100%',
+        zIndex: 10
+      },
+      componentTop: {
+        borderBottomWidth: '1px',
+        borderColor: '#E5E5E5',
+        borderRadius: '3px 3px 3px 3px',
+        borderStyle: 'solid',
+        borderWidth: '1px 1px 1px 1px',
+        position: 'relative',
+        padding: '10px 10px 0 10px'
+      },
+      componentTopOpen: {
+        borderBottomWidth: '0',
+        borderRadius: '3px 3px 0 0'
+      },
+      currentDay: {
+        backgroundColor: '#359BCF',
+        color: '#FFFFFF'
+      },
+      currentMonth: {
+        color: '#000000'
+      },
+      input: {
+        backgroundColor: '#FFFFFF',
+        border: 'none',
+        fontSize: '1em',
+        outline: 'none',
+        paddingBottom: '10px',
+        WebkitAppearance: 'none',
+        width: '80%',
+
+        ':focus': {
+          border: 'none',
+          boxShadow: 'none',
+          outline: 'none'
+        }
+      },
+      navIcon: {
+        color: '#000000',
+        cursor: 'pointer'
+      },
+      navLeft: {
+        position: 'absolute',
+        left: '0',
+        top: '50%',
+        transform: 'translateY(-50%)'
+      },
+      navRight: {
+        position: 'absolute',
+        right: '0',
+        top: '50%',
+        transform: 'translateY(-50%)'
+      },
+      scrim: {
+        position: 'fixed',
+        zIndex: 9,
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0
+      },
+      selectedDate: {
+        color: '#606060',
+        cursor: 'pointer',
+        fontSize: '1.2em',
+        fontWeight: 'bold',
+        verticalAlign: 'middle',
+        width: '100%',
+
+        ':hover': {
+          color: '#359BCF'
+        }
+      },
+      title: {
+        backgroundColor: '#666666',
+        color: '#f2f2f2',
+        textAlign: 'center',
+        padding: '7px 0px 7px 0px',
+        fontSize: '1em',
+        margin: '1px'
+      }
+    };
```

Comment:
Could we move this `styles` object outside of the React class? In the places we are using `props` to set styles, we can modify the styles object or use Radium to our advantage and do something like
```
style={[styles.calendarHeader, { borderBottomStyle: this.props.divideCells ? 'solid' : 'none'}]}
```
This would also make it so we don't have to pass the `styles` variable to the `_renderMonthlyTable` method.